site/app/templates/admin/LateDays.twig

```diff
@@ -77,4 +77,34 @@
     $("#user_id").autocomplete({
         source: {{ student_full|raw }}
     });
+    flatpickr("#datestamp", {
+            plugins: [ShortcutButtonsPlugin(
+                    {
+                        button: [
+                            {
+                                label: "Now"
+                            }, 
+                            {
+                                label: "End of time"
+                            }
+                        ],
+                        label: "or",
+                        onClick: (index, fp) => {
+                            let date;
+                            switch (index) {
+                                case 0: 
+                                    date = new Date();
+                                    break;
+                                case 1: 
+                                    date = new Date("01/01/98");
```

Comment:
Why are we using "01/01/98" here? the rest of the flatpickr "end of time" dates are 9998-01-01 which works fine.